README.md

```diff
@@ -2,7 +2,157 @@
 
 # Prelude
 
+ASON is a data oriented algorithm designed for compact and speedy storage of AssemblyScript objects in a binary format.
 
+There are many Serialization methods out there that can turn a conceptual object into a buffer of some kind, like JSON and protobuf. Each one of these methods has their benefits, and none of them can take advantage of runtime type information provided by the AssemblyScript compiler and the runtime type information.
```

Comment:
runtime type information provided by the AssemblyScript compiler ~and the runtime type information~